### PR TITLE
feat: Create new image build job for lte integ test

### PR DESCRIPTION
**Pull request by quentinDERORY**
focal image with pre-installed packages to run the integ tests

Signed-off-by: quentinDERORY <15911421+quentinDERORY@users.noreply.github.com>

### CODE REVIEW

File: packer/provision/local-docker.yaml [hunk 1]
```diff
@@ -2,3 +2,215 @@
...
+
+    - name: Link pyenv binary
+      file:
+        src: /home/jenkins/.pyenv/bin/pyenv
+        dest: /usr/local/bin/pyenv
+        state: link
+
+    - name: Install python 3.7.0
+      shell: pyenv install 3.7.0 -f
+      become: jenkins
```
Comment (jwagantall) [inline packer/provision/local-docker.yaml:176]: When testing this manually, I got the following error while trying to install the tools as jenkins

    openstack: the field 'become' has an invalid value (jenkins), and could not be converted to an bool.The error was: The value 'jenkins' is not a valid boolean.  Valid booleans include: 't', 1, '0', 0, 'off', 'no', 'true', 'y', 'false', 'f', 'on', 'yes', '1', 'n'. The value 'jenkins' is not a valid boolean.  Valid booleans include: 't', 1, '0', 0, 'off', 'no', 'true', 'y', 'false', 'f', 'on', 'yes', '1', 'n'

I ended up running all these with the global "become: true"

**Review by jwagantall — CHANGES_REQUESTED** [review]

File: packer/provision/local-docker.yaml (hunk 1 above)
Comment (jwagantall) [reply packer/provision/local-docker.yaml:176]: recheck

Comment (quentinDERORY) [reply packer/provision/local-docker.yaml]: Changed it to become_user

**Comment by quentinDERORY** [comment]
common-packer PR: https://github.com/lfit/releng-common-packer/pull/1

**Review by jwagantall — APPROVED** [review]

**Pull request merged**